src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.exception.DukeException;
+import duke.misc.Parser;
+import duke.misc.TaskList;
+
+/**
+ * Driver class to simulate the 'Annie' chat bot program.
+ */
+public class Duke {
+    private final TaskList tl;
```

Comment:
Constants should ideally be named using all caps separated with underscores to follow the coding standard. Could perhaps rename as TL?